lib/roby/interface/async/job_monitor.rb

```diff
@@ -86,23 +86,31 @@ def restart
                     interface.monitor_job(job_id)
                 end
 
+                def job_name
+                    action_name || task&.job_name
```

Comment:
I don't agree with this. If we have the ability to explictly give a name, it should take precedence over the default

Reply:
I did that because it changed some names that we are used to in syskit ide. The ui_rest job became `Wetpaint::Compositions::ui_rest` (or something like that), for example.

Reply:
I think it comes from here https://github.com//rock-core/tools-roby/blob/1fead3dc36d710626bd2290d43b4d530da4667ac/lib/roby/actions/task.rb#L40

It would be better to update this instead, for instance with:

```
def job_name
     arguments[:job_name] || action_name
end
```